src/auth/auth.service.ts

```diff
@@ -3,63 +3,85 @@ import { TokenPayload } from './interfaces/token-payload.interface';
 import { JwtService } from '@nestjs/jwt';
 import { v4 as uuidv4 } from 'uuid';
 import { AuthDTO } from './dto/auth.dto';
+import { AstraService } from '../astra/astra.service';
 
 @Injectable()
 export class AuthService {
-  //TODO move configCollection to database => own ConfigDataModule
-  private configCollection: { [id: string]: { knownClients: string[] } } = {};
-  constructor(private readonly jwtService: JwtService) {}
+  constructor(
+    private readonly jwtService: JwtService,
+    private readonly astraService: AstraService,
+  ) {}
 
-  public register(body: AuthDTO) {
+  public async register(body: AuthDTO) {
     const clientId = uuidv4();
     const { serverId, scopes } = body;
+    let tokens: string[] = [];
+    try {
+      tokens = await this.astraService
+        .get<string[]>('tokens', serverId, 'tokens')
+        .toPromise();
```

Comment:
Will this end point not create a new token every time? if so then can it added to the collection with a `create` rather than a `replace`?